src/Magnum/Math/Test/BezierTest.cpp

```diff
@@ -0,0 +1,81 @@
+/*
+    This file is part of Magnum.
+
+    Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016
+              Vladimír Vondruš <mosra@centrum.cz>
```

Comment:
You can add yourself here ;)

[-> Reference](https://github.com/mosra/magnum/blob/master/src/Magnum/Audio/Context.h#L8)